package.json

```diff
@@ -110,7 +109,7 @@
     "csv2json": "^1.4.2",
     "cypress": "13.5.0",
     "date-fns": "^2.28.0",
-    "dotenv": "^6.1.0",
+    "dotenv": "16.4.5",
```

Comment:
Had to upgrade dotenv to enable it parsing multiline environment variables strings